Lib/test/test_normalization.py

```diff
@@ -88,6 +88,11 @@ def run_normalization_tests(self, testdata):
                             NFKD(c3) == NFKD(c4) == NFKD(c5),
                             line)
 
+            self.assertTrue(is_normalized("NFC", c2))
+            self.assertTrue(is_normalized("NFD", c3))
+            self.assertTrue(is_normalized("NFKC", c4))
+            self.assertTrue(is_normalized("NFKD", c5))
```

Comment:
There should be some negative cases, too. Make sure the `MAYBE` case is being exercised.

Reply:
Increased coverage + confirmed that this is exercising the `MAYBE` path.

Reply:
Maybe add also tests when it returns False. If the function always returns True, the test still pass ;-)